src/app/app.component.html

```diff
@@ -0,0 +1,18 @@
+<mat-toolbar color="primary">
+  <a class="title" href="/">{{ title }}</a>
```

Comment:
That would be much better indeed. I had to get rid of the pages because of how the b2c demo environment and the web api was setup...  on my to-do list